swedish/somna-om/somna-om.md

```diff
@@ -1,4 +1,5 @@
 Du väljer att dra täcket över huvudet och somnar om,
 men mardrömmen fortsätter och du drömmer nu om att...
 
-[du äter en stor marshmallow...](../marshmallowen/marshmallowen.md)
\ No newline at end of file
+[du äter en stor marshmallow...](../marshmallowen/marshmallowen.md)
+[det är midsommar](../midsommar/midsommar.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.